nodewatcher/modules/devices/xiaomi/mi_router.py

```diff
@@ -0,0 +1,174 @@
+from django.utils.translation import ugettext_lazy as _
+
+from nodewatcher.core.generator.cgm import base as cgm_base, protocols as cgm_protocols, devices as cgm_devices
+
+
+class XiaomiMiMini(cgm_devices.DeviceBase):
+    """
+    Xiaomi Router Mini device descriptor.
+    """
+
+    identifier = 'mi-router-mini'
```

Comment:
Mi is Xiaomi brand name.
All of their WiFi product carry Mi name as well as their top range smartphones and other houshold appliances